API.md

```diff
@@ -1008,12 +1008,15 @@ curl http://localhost:<port>/irods-http-api/<version>/data-objects \
     --data-urlencode 'op=parallel_write_init' \
     --data-urlencode 'lpath=<string>' \ # Absolute logical path to a data object.
     --data-urlencode 'resource=<string>' \ # The root resource to write to. Optional.
+    --data-urlencode 'replica-number=<integer>' \ # The replica number of the replica to write to. Cannot be negative if specified. Optional.
```

Comment:
So... the client doesn't actually prevent providing a negative value (from what I can tell). Is the "Cannot" here more of a "Should not" due to https://github.com/irods/irods/issues/8617?

Reply:
Done.